tests/x11/rmt/rmt_feature.pm

```diff
@@ -0,0 +1,89 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Add rmt configuration test and basic configuration via
+#    rmt-wizard, test enable/disable products/repo, test rmt sync
+#    rmt mirror, test import SMT data to RMT
+# Maintainer: Yutao wang <yuwang@suse.com>
+
+use strict;
+use warnings;
+use testapi;
+use base 'x11test';
+use repo_tools;
+use utils;
+use x11utils 'turn_off_gnome_screensaver';
+
+sub run {
+    x11_start_program('xterm -geometry 150x35+5+5', target_match => 'xterm');
+    # Avoid blank screen since smt sync needs time
+    turn_off_gnome_screensaver;
+    become_root;
+    rmt_wizard();
+    # sync from SCC
+    rmt_sync;
+    # enable all modules of products
+    assert_script_run("rmt-cli p e SLES/15.2/x86_64 --all-modules");
+    assert_script_run("rmt-cli product list | grep sle-module-basesystem/15.2/x86_64");
+    assert_script_run("rmt-cli products enable --all-modules SLES/15");
+    assert_script_run("rmt-cli product list | grep sle-module-server-applications/15/aarch64");
+    assert_script_run("rmt-cli product list | grep sle-module-server-applications/15/ppc64le");
+    assert_script_run("rmt-cli product list | grep sle-module-server-applications/15/s390x");
+    assert_script_run("rmt-cli product list | grep sle-module-server-applications/15/x86_64");
+    assert_script_run("rmt-cli products disable 1946 1963 1967 1971 1982 1988 1992 1955 2075 1939 1950 1998 1973 1974 1976");
```

Comment:
It's hard for guest to understand the number, will the number changed later or will it keep stable?

Reply:
Same as above. We just test feature. The number will not change